hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-timelineservice-hbase-tests/pom.xml

```diff
@@ -104,6 +104,10 @@
           <groupId>com.sun.jersey</groupId>
           <artifactId>jersey-json</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>commons-logging</groupId>
+          <artifactId>commons-logging</artifactId>
+        </exclusion>
```

Comment:
Just to be clear, all the exclusions of commons-logging are required in this particular pom because it uses old hbase dependency, which internally pulls-in different version of hadoop-commons, with different commons-logging version.